Dockerfile.e2e

```diff
@@ -1,9 +1,9 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS build
+FROM mcr.microsoft.com/dotnet/sdk@sha256:e6b7bbd8daa77b072c6f1e0c18f531e63381cc6c8d75ee922fa9fd504749966e AS build
```

Comment:
should be this since it need upgrade to 5.0: 
```
docker inspect --format='{{index .RepoDigests 0}}' mcr.microsoft.com/dotnet/sdk:5.0
mcr.microsoft.com/dotnet/sdk@sha256:fa19559201c43bc8191c1a095670e242de80a23697d24f5a3460019958637c63
```